### DISCUSSION TITLE
mitmproxy  without mitm

### DISCUSSION THREAD

**Discussion by maxterrr** [mitmproxy/mitmproxy#5096] [Q&A]
I'd like to use mitmproxy mostly for analysis/visibility. Traffic and TLS secrets for the key log file would be captured by some other  applications. Wireshark/tshark has the option to output the decrypted traffic in multiple formats with "-T pdml|ps|psml|json|jsonraw|ek|tabs|text|fields" . Can you import one of those formats into mitmproxy? Or are there alternative ways to get decrypted traffic into mitmproxy (ideally live)?

**Comment by mhils** [comment]
I'm afraid we currently only support our own proprietary format. I would like to see support for HAR files next, which despite being a terrible format continues to be widely used. Other than that, I'm aware of [`httpreplay`](https://github.com/hatching/httpreplay), which does have a `pcap2mitm` command.
